utils/rpms/daos.spec

```diff
@@ -459,7 +468,13 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # set daos_firmware to be setuid root in order to perform privileged tasks
 %attr(4750,root,daos_server) %{_bindir}/daos_firmware
 
+%files daos_serialize
+%{_libdir}/libdaos_serialize.so
+
 %changelog
+* Wed Jul 14 2021 Danielle M. Sikich <danielle.sikich@intel.com> 1.3.103-4
+- Add DAOS serialization library that requires hdf5
```

Comment:
don't you need corresponding changes to debian/changelog?

Reply:
@frostedcmos I don't know this is my first time updating the daos.spec, and @brianjmurrell said these package changes looked good a few weeks ago.